charts/rode-collector-harbor/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "rode-collector-harbor.serviceAccountName" . }}
+  labels:
+    {{- include "rode-collector-harbor.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- end }}
```

Comment:
I don't think we really need rbac for this.  the collector isn't interacting with the kubernetes API in any way